games-engines/scummvm/scummvm-1.9.0.recipe

```diff
@@ -56,41 +58,56 @@ BUILD_REQUIRES="
 	devel:libogg$secondaryArchSuffix
 	devel:libpng$secondaryArchSuffix
 	devel:libSDL$secondaryArchSuffix
+	#devel:libsdl_net$secondaryArchSuffix #not needed yet
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-"
+	"
 
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
-	cmd:git
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:nasm
 	cmd:pkg_config$secondaryArchSuffix
-"
+	"
 
 BUILD()
 {
 	libtoolize --force --copy --install
 	CPPFLAGS=`freetype-config --cflags` ./configure --prefix=$prefix \
-	--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
-	--mandir=$manDir --docdir=$docDir \
-	--disable-debug --enable-release #--disable-all-engines
-	#--disable-all-engines can be used to only compile scummvm without
-	#engines, which makes it build faster, those that are still under
-	#construction can be built with --enable-all-engines for testing purposes
-
+		--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
+		--mandir=$manDir --docdir=$docDir \
+		--disable-debug --enable-release --disable-eventrecorder \
+		#--disable-all-engines #can be used to compile scummvm without \
+		#engines, which makes it build faster (to test), those that \
+		#are still under construction can be built with --enable-all-engines \
+		#for testing purposes
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 	mv $appsDir/scummvm $appsDir/ScummVM
-	addResourcesToBinaries $portDir/additional-files/scummvm.rdef \
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/scummvm.rdef > scummvm.rdef
```

Comment:
> I got the emails from you over github in regards to my master branch (but I can't see them here in the list) :)

Sorry, I was feeling guilty for having added so much crap to this PR, so I removed my comments after I fixed the conflict.
BTW, the steps to fix the conflict on your side **_would_** have been:

1) to revert the change about the renaming of the rdef file, probably with "git reset scummvm.rdef" and "git rm scummvm.rdef.in".
2) then merge master and rebase:
```
git fetch upstream master
git merge --no-edit upstream/master
git rebase upstream/master
```
3) Redo the rename and then push your local branch to your remote branch:
```
git mv scummvm.rdef scummvm.rdef.in
git commit --amend --no-edit scummvm.rdef scummvm.rdef.in
git push -f
```
This would only have worked, however, if you had your origin and upstream alias correctly set.